tsl/src/hypercore/hypercore_handler.c

```diff
@@ -3081,7 +3081,11 @@ static void
 hypercore_index_validate_scan(Relation compressionRelation, Relation indexRelation,
 							  IndexInfo *indexInfo, Snapshot snapshot, ValidateIndexState *state)
 {
-	FEATURE_NOT_SUPPORTED;
+	ereport(ERROR,
+			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+			 errmsg("concurrent index builds not supported"),
+			 errdetail("Hypercore table access method does not support concurrent index builds."),
```

Comment:
I guess we can skip errdetail. It doesn't add anything.

If you want to keep, it should at the very least print the name of the relation that doesn't support it. It might be useful if this recurses on a hypertable and you only have one Hypercore chunk.

Reply:
Btw, this is what the errors in postgresql look like. Maybe we should follow that pattern:

```
if (concurrent &&
		IsCatalogRelation(heapRelation))
		ereport(ERROR,
				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
				 errmsg("concurrent index creation on system catalog tables is not supported")));

	/*
	 * This case is currently not supported.  There's no way to ask for it in
	 * the grammar with CREATE INDEX, but it can happen with REINDEX.
	 */
	if (concurrent && is_exclusion)
		ereport(ERROR,
				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
				 errmsg("concurrent index creation for exclusion constraints is not supported")));

```

Reply:
Removed the detailed message and the hint and changed the message.